lib/LaTeXML/Package/enumitem.sty.ltxml

```diff
@@ -65,7 +65,11 @@ DefKeyVal('enumitem', 'widest',       'UndigestedKey');
 DefKeyVal('enumitem', 'beginpenalty', 'Number');
 DefKeyVal('enumitem', 'midpenalty',   'Number');
 DefKeyVal('enumitem', 'endpenalty',   'Number');
-DefKeyVal('enumitem', 'noitemsep',    'UndigestedKey');
+DefKeyVal('enumitem', 'noitemsep',    '', 'true');
+DefKeyVal('enumitem', 'nolistsep',    '', 'true');
+
+DefMacroI('\KV@enumitem@nolistsep@default', undef, Tokens());
+DefMacroI('\KV@enumitem@noitemsep@default', undef, Tokens());
```

Comment:
with the defaults given, you shouldn't need these last two macros

Reply:
Thanks for the mini tutorial. Tested, and it is exactly as you say.

Feels good to understand these bits a little better on my part.